lib/LaTeXML/Post.pm

```diff
@@ -447,7 +447,7 @@ sub convertXMTextContent {
   foreach my $node (@nodes) {
     if ($node->nodeType == XML_TEXT_NODE) {
       my $string = $node->textContent;
-      $string =~ s/^\s+/$NBSP/; $string =~ s/\s+$/$NBSP/;    # should we???
```

Comment:
It's Presentation MathML's mtext that is relevant.

Looking at that ticket, I am not sure if @kohlhase is referring to the linebreaks (intended to be "ignorable whitespace") inserted by formatting, or the non-break-spaces (```&#xA0;```).  Again, libxml2 is inserting the "ignorable whitespace" for formatting without knowing the schema of OpenMath; since om:FOREIGN accepts plain text, there really should *not* be any formatting whitespace inserted.

I don't find any clear indication of whether om:FOREIGN has any special rules for whitespace. OTOH, since the FOREIGN element says encoding="mtext", one could argue that it should use m:mtext's whitespace rules? :>

Perhaps the method that's defined in Post.pm isn't appropriate for OpenMath? I don't know, and am not yet clear about what's desired for OpenMath.  As it is, I'm apparently going to have to do some fixing of that method even for MathML!